providers/SCConfigurationProvider.js

```diff
@@ -2,7 +2,17 @@
 
     'use strict';
 
-    angular.module('soundcloudify.core').provider('SCConfiguration', SCConfigurationProvider);
+    try{
+        angular.module('soundcloudify.core')
+            .provider('SCConfiguration', SCConfigurationProvider);
+    }
+    catch (err){ /*ignore*/ }
+
+    try{
+        angular.module('soundcloudify.background')
+            .provider('SCConfiguration', SCConfigurationProvider);
+    }
```

Comment:
@valevalorin : you can add the core module in to the background page by adding these scripts tag into background.html

```
<script src="scripts/core/app.js"></script>

<script src="scripts/core/controllers/MainController.js"></script>
<script src="scripts/core/controllers/DiscoveryController.js"></script>
<script src="scripts/core/controllers/PlaylistController.js"></script>
<script src="scripts/core/controllers/PlaylistViewController.js"></script>
<script src="scripts/core/controllers/ChartsController.js"></script>
<script src="scripts/core/controllers/ChartsViewController.js"></script>
<script src="scripts/core/controllers/NowPlayingController.js"></script>
<script src="scripts/core/controllers/SearchController.js"></script>

<script src="scripts/core/directives/scdPlayerDirective.js"></script>
<script src="scripts/core/directives/loadingDirective.js"></script>
<script src="scripts/core/directives/trackTitleDirective.js"></script>
<script src="scripts/core/directives/toggleButtonDirective.js"></script>
<script src="scripts/core/directives/playlistMenuDirective.js"></script>
<script src="scripts/core/directives/trackListDirective.js"></script>
<script src="scripts/core/directives/playingTrackAwareDirective.js"></script>
<script src="scripts/core/directives/configurationDirective.js"></script>
<script src="scripts/core/directives/rightClickDirective.js"></script>

<script src="scripts/core/providers/Paginator.js"></script>
<script src="scripts/core/providers/SCConfigurationProvider.js"></script>

<script src="scripts/core/filters/scArtworkFilter.js"></script>
<script src="scripts/core/filters/prettyTimeFilter.js"></script>
<script src="scripts/core/filters/translateFilter.js"></script>

<script src="scripts/core/services/StorageService.js"></script>
<script src="scripts/core/services/Messaging.js"></script>
<script src="scripts/core/services/CategoryService.js"></script>
<script src="scripts/core/services/SearchService.js"></script>
<script src="scripts/core/services/SearchSingleton.js"></script>
<script src="scripts/core/services/SuggestionService.js"></script>
<script src="scripts/core/services/NowPlayingService.js"></script>
<script src="scripts/core/services/PlaylistService.js"></script>
<script src="scripts/core/services/PlaylistImporter.js"></script>
<script src="scripts/core/services/StarService.js"></script>
<script src="scripts/core/services/TrackAdapter.js"></script>
<script src="scripts/core/services/GATracker.js"></script>
<script src="scripts/core/services/LastFMAuthentication.js"></script>
<script src="scripts/core/services/UserService.js"></script>
<script src="scripts/core/services/HttpRequestInterceptor.js"></script>
<script src="scripts/core/services/SyncService.js"></script>
<script src="scripts/core/services/MigrationService.js"></script>
```

Just get rid of the service you don't need. But the important file here is `core/app.js` as it define the module.